CHANGELOG.md

```diff
@@ -1,14 +1,21 @@
 # Changelog
```

Comment:
The CHANGELOG.md should not be part of the review. Pull the latest changes from the master branch and then rebase your current branch.

```
git pull origin master
git rebase master
git push origin '<The name of your branch>' --force
```